backend/server.js

```diff
@@ -1,9 +1,43 @@
-const express = require("express");
+const express = require('express');
 const app = express();
 const port = process.env.PORT || 5000;
+const bodyParser = require('body-parser');
+const totp = require("totp-generator");
+const { v4: uuid } = require('uuid');
 
 app.listen(port, () => console.log(`Listening on port ${port}`));
 
+app.use(bodyParser.json());
+
+app.get('/', (req, res) => {
+    res.send('Hello World!');
+});
+
+app.get('/signup', (req, res) => {
+    const userId = generateUserId();
+    const sharedSecret = userId.split("-")[4];
```

Comment:
should the secret be a separate string rather than part of the userId?

Reply:
Definitely agree, this is a hacky way of having a validation work without any storage needed on the backend.

Reply:
if thats a future change and this is just poc then i approve

Reply:
ah yeah i just fully missed the PR description